### ISSUE TITLE
500+ star, release the ProMax version with tile and inpainting!!!

### ISSUE THREAD

**Issue by T8mars** [xinsir6/ControlNetPlus#12]
thanks for your works,I will continue to promote it for you, hoping to reach 1000 stars as soon as possible

**Comment by xinsir6** [comment]
Yes, the promax model is on the way, I am busy with it recently. I will release it in the weekeed(07/13, 07/14). Thank you for your attention. Happy to hear your feedback.

**Comment by T8mars** [comment]
My pleasure,I will promote the github going on,thanks for you nice work.
